portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,10 +35,99 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+   // private List<String> messages;
```

Comment:
I think some of these changes are leftover from your last PR? I left a couple comments over there, so I'm going to leave them alone here.